getting-started/choosing-a-development-host/mac-osx.md

```diff
@@ -1,4 +1,69 @@
 # Mac OSX
 
-Under development
+You can run Rocket.Chat for development on  Mac OSX. The following instructions has been tested on  M1 Macs.
 
+1. Install Meteor 
+
+    Currently Meteor don’t support Apple M1 native binaries as the latest meteor release uses Node.js 14 which doesn’t have support for it until now.
+
+    You can use Rosetta and enter the below command.
+
+    `arch -x86_64 npm install -g meteor`
+
+    or select Terminal in the Applications folder, press CMD(⌘)+I and check the “Open using Rosetta” option then using this command.
+
+    `npm install -g meteor`
+
+    (If your user doesn’t have permission to install global binaries, and you need to use sudo, it’s necessary to append –unsafe-perm to the below command:
+     `sudo npm install -g meteor --unsafe-perm` )
+     
+      There is no need to install `node` or `npm`, as meteor already includes them. Verify by:
+      
+      `meteor node -v`
+      
+      `meteor npm -v`
+
+   
+2) Fork the repository ` https://github.com/RocketChat/Rocket.Chat.git` and then clone the forked repository in your system.
+
+3) Install Modules
+
+    `cd Rocket.Chat`
+
+    `meteor npm install`
```

Comment:
Previous comment. **Currently** we need meteor 2.2 for a successful build :( This will most likely fail. Or cause some nasty runtime exceptions down the road if you get to that point *somehow*.

Reply:
to prevent this a previous step can be added:
`meteor --version`

this will make sure the correct Meteor and Node versions are installed for the project.

Reply:
When i run `meteor --version`. This is what the output is 

`This project uses Meteor 2.2, which isn't available on this platform. To work with this app on all supported platforms, use meteor update --release METEOR@2.5.1 to pin this app to the newest compatible release.`